frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -122,7 +122,6 @@ describe('MapProperties View', () => {
   });
 
   afterEach(() => {
-    window.ResizeObserver = ResizeObserver;
     jest.restoreAllMocks();
```

Comment:
Not really sure why this was causing an error: `TypeError: window.ResizeObserver is not a constructor`